models/message.go

```diff
@@ -108,25 +108,20 @@ func CreateMessage(fields *Message) error {
 // URLs, and then passing the body through Blackfriday for additional parsing
 // and finally Bluemonday for sanitization.
 func ParseMessage(message *Message) string {
-	body := message.Body
+	messageCopy := *message
```

Comment:
This looks a little strange. It's not actually copying the message. Do you need to make a deep copy? I wonder if we should approach this differently, do we need to mutate the internal state of `Message`?